config.cc

```diff
@@ -52,6 +52,13 @@ namespace
       result = QDir::fromNativeSeparators( QString::fromWCharArray( _wgetenv( L"APPDATA" ) ) );
     #else
       char const * pathInHome = ".goldendict";
+      #if QT_VERSION >= QT_VERSION_CHECK( 5, 0, 0 )
+        if ( !result.exists( pathInHome ) )
+        {
```

Comment:
Maybe add a comment here to explain the purpose of this conditional directory replacement.

Reply:
Done, and I added more detailed description in commit message.